code/modules/food_and_drink/plants.dm

```diff
@@ -1198,24 +1114,23 @@ ABSTRACT_TYPE(/obj/item/reagent/containers/food/snacks/plant)
 	planttype = /datum/plant/veg/onion
 	food_color = "#FF9933"
 	food_effects = list("food_bad_breath")
+	sliceable = TRUE
+	slice_product = /obj/item/reagent_containers/food/snacks/onion_slice
+	slice_amount = 4
+	slice_suffix = "ring"
+
+	New()
+		..()
+		src.slice_amount = rand(3, 5) //keeps the previous random amount of slices onions had
```

Comment:
```suggestion
		src.slice_amount = rand(3, 5)
```

Reply:
To be honest, in hindsight, it should have been a different PR. I originally wanted to go over most produce types and get rid of any chems that get added by this proc when they are harvested.